docs/resources/generated-error-pages/error-codes-inventory.rst.inc

```diff
@@ -492,13 +492,163 @@ PACKAGE_UPLOAD_REJECTED
 
 
 
-2.2.1.1. ParticipantErrorGroup / LedgerApiErrors / AdminServices / UserManagementServiceErrors
+2.2.1.1. ParticipantErrorGroup / LedgerApiErrors / AdminServices / PartyManagementServiceErrorGroup
 ===================================================================================================================
 
 
 
 
 
+.. _error_code_CONCURRENT_PARTY_DETAILS_UPDATE_DETECTED:
+
+CONCURRENT_PARTY_DETAILS_UPDATE_DETECTED
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: Concurrent updates to a party can be controlled by optionally supplying an update request with a resource version. A party's resource version can be obtained by reading the party on the Ledger API. There was attempt to update a party using a stale resource version indicating a different processes had updated the party earlier.
+
+    **Category**: ContentionOnSharedResources
+
+    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status ABORTED including a detailed error message
```

Comment:
I created a separate PR to improve the conveyance wording, because we generate them from the same templates for all the error codes.
PR: https://github.com/digital-asset/daml/pull/15043

(I'll be also marking all conveyance related comments in this PR as resolved.)